### PR TITLE
Clean up some Rubocop offenses

### PR DESCRIPTION
**Pull request by errinlarsen**
##### Modified configuration for Rubocop a bit:

* ignore vendored files
* removed `Blocks` section
* removed RegexpLiteral section
* added MultilineOperationIndentation section

##### Cleaned up a few Rubocop offenses

* ~~Align operands in multi-line assignments~~
* _Indent_ operands in multi-line operation chains
* Modify arguments to indicate if they won't be used
* Remove parallel assignment
* Add a space inside curly braces for blocks
* Remove extra line from block body beginning
* Use Array#size instead of Array#count
* Replace {} with do/end for procedural block
* Remove semicolon as expression terminator

###### _Edit (5/26/2015):_

  * _`added MultilineOperationIndentation section`_
  * _`Indent operands in multi-line operation chains`_

### CODE REVIEW

File: benchmarks/setup.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@ def tags
 
 def users_with_combined_tasks
   @users_with_combined_tasks ||= rom.relation(:users).as(:user_with_combined_tasks)
-    .combine(rom.relation(:tasks).for_users)
+                                 .combine(rom.relation(:tasks).for_users)
```
Comment (solnic) [inline benchmarks/setup.rb]: We decided not to do that. Not sure why rubocop is still configured to report that as an offense

Comment (errinlarsen) [inline benchmarks/setup.rb]: I can fix this, and update the rubocop config to stop reporting these offenses.

Want me to create a new pull-request? add a commit to this one?

Comment (solnic) [inline benchmarks/setup.rb]: Add a commit to this one please

**Comment by solnic** [comment]
ugh, why have I missed that one? now it doesn't merge cleanly ;(

**Comment by vrish88** [comment]
@errinlarsen do you mind fixing up the conflicts? If not I've got some time this week and I could fix it up.

**Comment by errinlarsen** [comment]
yup. I'll get it fixed up today.

**Comment by errinlarsen** [comment]
Closing this pull-request; merge conflicts are fixed in #278.

**Pull request closed without merging**